Source/EasyNetQ/ServiceRegisterExtensions.cs

```diff
@@ -63,7 +65,9 @@ public static class ServiceRegisterExtensions
             .TryRegister<IRpc, DefaultRpc>()
             .TryRegister<ISendReceive, DefaultSendReceive>()
             .TryRegister<IScheduler, DeadLetterExchangeAndMessageTtlScheduler>()
-            .TryRegister<IBus, RabbitBus>();
+            .TryRegister<IBus, RabbitBus>()
+            .TryRegister(typeof(ILogger<>), typeof(Logger<>))
```

Comment:
Just for my understanding: is it a common approach try to register a default logging implementation inside a library?

Reply:
@zidad Ok, let's proceed in this way at least for now.